src/tokenList.json

```diff
@@ -102,5 +102,43 @@
     "website": "https://www.synthetix.io/ ",
     "description": "Tracks the price of a single US Dollar (USD). This Synth always remains constant at 1.",
     "rinkeby_faucet": "https://rinkeby.etherscan.io/address/0xDFd3CAde426Fc6A014fd2e0E1Cb3158F972A1A0D. Instructions: https://github.com/Synthetixio/synthetix/issues/382#issuecomment-579518740"
+  },
+  {
+    "id": 10,
+    "name": "Synthetix Network",
+    "symbol": "SNX",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0xC011A72400E58ecD99Ee497CF89E3775d4bd732F",
+      "4": "0x322A3346bf24363f451164d96A5b5cd5A7F4c337"
+    },
+    "website": "https://www.synthetix.io",
+    "description": "The Synthetix Network Token (SNX) gets staked as collateral to back Synths and entitles stakers to receive fees generated by Synth trades on Synthetix.Exchange",
+    "rinkeby_faucet": "https://rinkeby.etherscan.io/address/0xDFd3CAde426Fc6A014fd2e0E1Cb3158F972A1A0D. Instructions: https://github.com/Synthetixio/synthetix/issues/382#issuecomment-579518740"
+  },
+  {
+    "id": 11,
+    "name": "Synth sETH",
+    "symbol": "sETH",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0x5e74C9036fb86BD7eCdcb084a0673EFc32eA31cb"
+    },
+    "website": "https://www.synthetix.io",
+    "description": "Tracks the price of Ether (ETH) through price feeds supplied by an oracle",
+    "rinkeby_faucet": null
+  },
+  {
+    "id": 12,
+    "name": "OWL",
+    "symbol": "OWL",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0x1A5F9352Af8aF974bFC03399e3767DF6370d82e4",
+      "4": "0xa7D1C04fAF998F9161fC9F800a99A809b84cfc9D"
+    },
+    "website": "https://gnosis.io/tokens",
+    "description": "OWL can be used to pay 1 USD in fees on Gnosis platforms",
+    "rinkeby_faucet": null
```

Comment:
Maybe devops has a rinkeby faucet we can use?

Reply:
I believe @fleupold (or somebody in the smart contracts) creating a smart contract to exchange DAI for OWL (only in DAI->OWL direction)

This would be enough to get some OWL, although a faucet for GNO and OWL is sth the company would need. It was talked many times, but no-one took ownership of that one. Looks like a good eco-friday challenge (to reuse an existing faucet for our needs)

Reply:
Do you mean for rinkeby or mainnet? The DAI -> OWL Converter was supposed to work for mainnet.

Reply:
The important one is mainnet. But if it's "free" we can deploy the same contract in rinkeby, for testing, and also for allowing to get some OWL there (of course, it can abused, but I think it's fine)

Reply:
My meaning for faucets in this file only refer to rinkeby, so anyone testing it can have `play money`

Reply:
I added an order selling OWL for DAI that should be fillable at 1:1 (100OWL for now). So if you ever need mainnet OWL feel free to trade it on dƒusion